### DISCUSSION TITLE
Problem with opening PowerShell at directory with special char in path

### DISCUSSION THREAD

**Discussion by keenoptical** [PowerShell/PowerShell#24704] [Q&A] [answered]
Hi there. I've got a problem with opening PowerShell in a directory whose path contains $ sign. Error is "Can't find the path because it doesn't exist". Previously it worked correctly. This probably started appearing with the latest Windows update. How to fix it?

<img width="863" alt="475745457" src="https://github.com/user-attachments/assets/70ffb08b-1559-4d52-870a-3d1017e5755b" />

**Comment by SamErde** [comment]
The `Set-Location` function shown in your error is using the **LiteralPath** parameter. This parameter does not take wildcards or variables -- it uses the path exactly as typed, including special characters. Given that fact, it should work if your path does actually include a '$' symbol. Are there any quotes around the path string? I would try wrapping it in *single* quotes and see if that helps.

**Reply by keenoptical** [reply]
There is no way to wrap path into single quotes. I don't run the `Set-Location` function by myself. This error happens after opening PowerShell in a specific folder by menu item "Open here".

![open-powershell-window-here](https://github.com/user-attachments/assets/086983fd-7b5e-4ae3-97ea-7671dbca5d38)

**Comment by keenoptical** [comment, marked as answer]
Problem was solved by modifying `HKEY_CLASSES_ROOT\Directory\Background\shell\Powershell\command` registry section. The default key in this section contains run script command. I've changed `"%V"` interpolation to `""%V""` and finally it works correctly. 

<img width="858" alt="33345" src="https://github.com/user-attachments/assets/19ea3c2e-851c-46e5-9c68-c102333f83d4" />
